modules/kafka-cluster/inputs.tf

```diff
@@ -30,5 +23,20 @@ variable "region" {
   description = "The cloud service provider region where the Kafka cluster is running - see supported list - https://docs.confluent.io/cloud/current/clusters/regions.html#cloud-providers-and-regions"
   type        = string
   default     = "asia-southeast2"
+}
 
+variable "cluster_type" {
+  description = "The value can be either basic, standard, and dedicated."
+  type        = string
+  default     = "basic"
+}
+variable "cku" {
+  description = "The number of Confluent Kafka Units (CKUs) for Dedicated cluster types. The minimum number of CKUs for SINGLE_ZONE dedicated clusters is 1 whereas MULTI_ZONE dedicated clusters must have more than 2 CKUs."
+  type        = number
+  default     = 1
+}
+variable "network_id" {
```

Comment:
`network_id` is relevant only for dedicated clusters, right? So maybe we can add that to the description.

Reply:
added and also prefixed with dedicated